### ISSUE TITLE
[CoE Starter Kit - BUG]  App are not aligned between Coe and Power Apps

### ISSUE THREAD

**Issue by JackSocini** [microsoft/coe-starter-kit#2315]
### Describe the issue

Hi, what does 250 mean? Does that mean it only shows 250 items?
Thanks

![flow-question](https://user-images.githubusercontent.com/101250592/160146277-5b4a635c-8963-445d-84ef-fe3668d81585.jpg)

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.34

### What app or flow are you having the issue with?

Admin | Sync Template v3 (Apps)

**Comment by Jenefer-Monroe** [comment]
Correct, you can see what these things mean by looking at the [connector information](https://docs.microsoft.com/connectors/powerappsforadmins/#get-apps-as-admin) as shown here:
![image](https://user-images.githubusercontent.com/89584709/160150427-e81854ff-0177-48c4-8e34-0e41b8d8de50.png)


We get more apps that 250 by using the pagination setting for the action
![image](https://user-images.githubusercontent.com/89584709/160150689-d323ff18-0819-43e7-860e-80255bb54ddf.png)

**Comment by Jenefer-Monroe** [comment]
Note that if you are only seeing a very small number of objects, you may need to do a [full inventory run](https://docs.microsoft.com/power-platform/guidance/coe/faq#running-a-full-inventory)

And if you are seeing small differences between the different sources (eg powershell v CoE) then it could be either that you do not have all the clean up flows running, or you may just be seeing some of the expected differences. 
For this please validate you have all the flows in the [inventory setup](https://docs.microsoft.com/power-platform/guidance/coe/setup-core-components) running and passing. 
And see also this documentation on [expected differences between powershell, ppadmin, and coe](https://docs.microsoft.com/power-platform/guidance/coe/limitations#inventory-and-telemetry-differences-between-powershell-power-platform-admin-center-and-coe-starter-kit)

**Comment by JackSocini** [comment]
I do full inventory but nothing change.. where I can understand if it work correctly?

**Comment by Jenefer-Monroe** [comment]
I dont understand the issue you are seeing, so I cannot assist until you provide more information. Can you please provide more detail on what you are and are not seeing? 

- Do you see all the environments? 
- Do you see only a few objects and expect many? 
- Do you see only objects to which you have permissions?

**Comment by JackSocini** [comment]
1. I see all environment.
2. I doesn't see item on CoE Toolkit that I see on environment
3. I'm power platform admin

**Comment by Jenefer-Monroe** [comment]
My assumption then is that you are seeing the small differences in count as described here: [expected differences between powershell, ppadmin, and coe](https://docs.microsoft.com/power-platform/guidance/coe/limitations#inventory-and-telemetry-differences-between-powershell-power-platform-admin-center-and-coe-starter-kit)

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team
